templates/application-keda.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: keda
+  annotations:
+    argocd.argoproj.io/sync-wave: "4"
+spec:
+  destination:
+    name: "in-cluster"
+    namespace: glueops-core-keda
+  source:
+    repoURL: 'https://kedacore.github.io/charts'
+    targetRevision: 2.10.2
+    chart: keda
+    helm:
+      values: |-
+        resources:
+          operator:
+            requests:
+              cpu: 100m
+              memory: 100Mi
+            limits:
+              cpu: 1
+              memory: 1000Mi
+        prometheus:
+          metricServer:
+            enabled: true
+            serviceMonitor:
+              enabled: true
+          operator:
+            enabled: true
+            port: 8080
+            serviceMonitor:
+              enabled: true
+            # prometheusRules:
```

Comment:
@debu99 can you add these alerts into this helm chart https://github.com/GlueOps/platform-helm-chart-alerts ?

Reply:
@debu99 Once you have pulled out the alerts feel free to bump this chart to: 0.14.0

https://github.com/GlueOps/platform-helm-chart-platform/blob/main/Chart.yaml#L5